doc/pod-creation-example.md

```diff
@@ -0,0 +1,88 @@
+# Example - Allocating MAC Addresses to a Pod
+
+## Create a network-attachment-definition:
+
+This example will use [ovs-cni](https://github.com/kubevirt/ovs-cni/).
+
+```yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ovs-conf
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: ovs-cni.network.kubevirt.io/br1
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "name": "ovs-conf",
+      "plugins" : [
+        {
+          "type": "ovs",
+          "bridge": "br1",
+          "vlan": 100
+        },
+        {
+          "type": "tuning"
+        }
+      ]
+    }'
+```
+
+**note** the project supports only json configuration for `k8s.v1.cni.cncf.io/networks`, network list will be ignored
+
+## Allocate a MAC Addresses to a Pod
+
+Create the Pod definition:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: samplepod
+  annotations:
+    k8s.v1.cni.cncf.io/networks: '[{ "name": "ovs-conf"}]'
+spec:
+  containers:
+  - name: samplepod
+    image: quay.io/schseba/kubemacpool-test:latest
+    imagePullPolicy: "IfNotPresent"
+```
+
+Check Pod deployment:
+
+```yaml
+Name:               samplepod
+Namespace:          default
+Priority:           0
+PriorityClassName:  <none>
+Node:               node01/192.168.66.101
+Start Time:         Thu, 14 Feb 2019 13:36:23 +0200
+Labels:             <none>
+Annotations:        
+  k8s.v1.cni.cncf.io/networks: [{"name":"ovs-conf","namespace":"default","mac":"02:00:00:00:00:02"}]
+                    k8s.v1.cni.cncf.io/networks-status:
```

Comment:
This indentation does not look right. Also, there is a lot of noise around this. And there is no command telling me how to get this output.

Could we change it to only get the network annotation specifically, to look at the MAC?

As a user I'd appreciate simple copy paste example and the output should be short, so I don't need to think hard what to look for (before reading the example below)

Reply:
I agree. 
Do you think I should leave both `k8s.v1.cni.cncf.io/networks` and `k8s.v1.cni.cncf.io/networks-status` as they both contain the allocated MAC, or is `k8s.v1.cni.cncf.io/networks` sufficient?